### ISSUE TITLE
[bug discovery] DR | How is EMMS/Data Dimensions handling upstream failures?

### ISSUE THREAD

**Issue by saderagsdale** [department-of-veterans-affairs/va.gov-team#70492]
## What happened?

- Roughly ~500 Evidence files for NOD and SC generated errors after Lighthouse passed the file to Centralized Mail Portal. 
- Error types include:
    * `max page Dimensions(inches) exceeded`
    * `Unsupported File type (only PDFs are accepted)`
    * `Unidentified Mail`
    * `Non-string values for keys: zipCode`
    * `ZIP Code must be 5 or 9 digits`
    * `Upstream status: Errors: Batch Submitted with all blank Images.;`
    * `Invalid PDF content, part content`
    * `Max File size(bytes) exceeded`
    * `ERR-EMMS-FAILED, Corrupted File detected.`

## What do we need to know?
Questions for EMMS/Data Dimensions:

1. Is there documentation that you can share to help us create a list of all potential errors we could receive? We're looking for:

- Errors definitions (What does it mean?)
- Impact of the error (What happens to the data when this error is flagged?)
- Validation criteria (What is the criteria for hitting this error?)
- Grouping (Are any error types bundled into a single response?)
- Current handling processes (Does a human or system step in to fix what went wrong and make the claim whole?)
- Polling (Is the error returned to Lighthouse?)
- Timing (How long does it take for Lighthouse to get the error?)

2. We are currently getting these errors below from upstream services. Is EMMS/Data Dimensions aware of any other errors that are happening or that we should be receiving?

## Action items
- [x] Compile and send questions to EMMS/Data Dimensions contact (John Clark)
- [x] Confirm if EMMS/Data Dimensions has intervention processes in place for each error type
- [x] Document findings in the [Sharepoint doc](https://dvagov.sharepoint.com/:x:/s/BenefitsDecisionReviewTeam/EY5P1JkGhZdIlG2vVx7XAH4Bdg8Od0yhMcswgTvyon_WQg?e=rSq8Ss)

## Slack threads and background context
- Link to Slack [thread 1](https://dsva.slack.com/archives/CKUCFQAHJ/p1697134097537579), [thread 2](https://dsva.slack.com/archives/CKUCFQAHJ/p1698238297544329), [thread 3](https://dsva.slack.com/archives/C053U7BUT27/p1700233242312199)

## Definition of done
- [ ] Action items above are completed
- [ ] We know if EMMS/Data Dimensions has a process in place to handle each error and make the submission whole

## Out of scope
- Coding solutions

**Comment by HeatherWidmont** [comment]
@saderagsdale got this scheduled for Thursday afternoon (Eugene included)

**Comment by saderagsdale** [comment]
Annie leading, Eugene supporting. Eugene will transition in Annie.

**Comment by HeatherWidmont** [comment]
@anniebtran is waiting on folx on the VA side to get us more info

**Comment by saderagsdale** [comment]
We got a list of error messages from the MMS team yesterday. Closing this ticket so we can move on to the next one in the epic.
